solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,180 @@
+        costumers_dictionary (dictionary): Contains bank costumer data.
+
+    Returns:
+        True for success withdrawal action, False otherwise.
+
+    """
+    balance_now = float(costumers_dictionary[costumer_indent][
+                            COSTUMER_BALANCE_PLACE_IN_DICTIONARY])
+
+    if 0 < amount_to_withdrawal <= balance_now:
+        new_balance = balance_now - amount_to_withdrawal
+        costumers_dictionary[costumer_indent][
+            COSTUMER_BALANCE_PLACE_IN_DICTIONARY] = str(new_balance)
+        return True
+
+    return False
+
+
+def cash_deposit(costumer_indent, amount_to_deposit, costumers_dictionary):
+    """Deposit to the bank some of money.
+
+    Args:
+        costumer_indent (string): Costumer id.
+        amount_to_deposit (float): Amount the costumer wants to deposit.
+        costumers_dictionary (dictionary): Contains bank costumer data.
+
+    Returns:
+        True for success deposit action, False otherwise.
+
+    """
+    if amount_to_deposit > 0:
+        balance_now = float(costumers_dictionary[costumer_indent][
+                                COSTUMER_BALANCE_PLACE_IN_DICTIONARY])
+        new_balance = balance_now + amount_to_deposit
+        costumers_dictionary[costumer_indent][
+            COSTUMER_BALANCE_PLACE_IN_DICTIONARY] = str(new_balance)
+        return True
+
+    return False
+
+
+def change_password(costumer_indent, new_password, costumers_dictionary):
+    """Changing the costumer's ATM password.
+
+    Args:
+        costumer_indent (string): Costumer id.
+        new_password (string): The new password the costumer wants to
+                               change to.
+        costumers_dictionary (dictionary): Contains bank costumer data.
+
+    Returns:
+        True for success changing the password, False otherwise.
+
+    """
+    if len(new_password) == 4 and new_password.isdigit():
+        costumers_dictionary[costumer_indent][
+            COSTUMER_PASSWORD_PLACE_IN_DICTIONARY] = new_password
+        return True
+
+    return False
+
+
+def fill_dictionary(costumers_dictionary):
+    """Filling dictionary with bank customer data.
+
+    Args:
+         costumers_dictionary (dictionary): An empty dictionary.
+
+    Returns:
+        A full dictionary of bank costumer data, or an exception if the
+        file path is not correct.
+
+    """
+    try:
+        with open(sys.argv[ATM_FILE], 'r') as atm_file:
+            for line in atm_file:
+                a, *b = line.split()
+                costumers_dictionary[a] = b
+            return costumers_dictionary
+
+    except FileNotFoundError:
+        return "Your file path is incorrect. Please check it."
+
+
+def main():
+    costumers_dictionary = {}
+    fill_dictionary(costumers_dictionary)
+    print("Welcome to bank!\n")
+    costumer_indent = input("Enter your id: ")
+    costumer_password = input("Enter your password: ")
+    not_finish_actions = True
+
+    while not_finish_actions:
```

Comment:
At the moment once the user finish, you program terminates instead of letting the next customer work (the task says terminate the program when -1 entered)